README.md

```diff
@@ -198,16 +194,16 @@ $ make kubeconfig
 Let's test accessing the Kubernetes API and list the Kubernetes cluster nodes:
 
 ```bash
-$ ./kubectl get nodes
+$ ./kubectl --context devkubernetes01 get nodes
 NAME                                          STATUS    ROLES     AGE       VERSION
-kube-node-0-kubelet.kubernetes.mesos          Ready     <none>    3m        v1.10.7
-kube-node-public-0-kubelet.kubernetes.mesos   Ready     <none>    2m        v1.10.7
+kube-node-0-kubelet.kubernetes.mesos          Ready     <none>    3m        v1.12.1
```

Comment:
This list of nodes isn't updated, we should see the kube-control-plane node ` kube-control-plane-0...`